### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by egernst**
Add code of conduct based on Contributor Covenant v1.4

Signed-off-by: Eric Ernst <eric.ernst@intel.com>

### CODE REVIEW

**Comment by egernst** [comment]
@amshinde @jodh-intel

**Comment by amshinde** [comment]
@egernst Can you add a subsystem in the commit message?

**Comment by egernst** [comment]
@amshinde done

**Comment by egernst** [comment]
we're looking @ creating a mailing list for CoC issues, so this PR will need updating

**Comment by egernst** [comment]
Updated - should be okay to merge now, as far as I'm concerned..

**Comment by iphutch** [comment]
@egernst Can you apply line length changes? 78 chars is standard.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
```
Comment (iphutch) [inline CODE_OF_CONDUCT.md]: Fix this indentation and we are good to go.

**Comment by egernst** [comment]
I ended up pulling direct from the convenant website and then squashing the commit history.  This renders fine for me now; let me know if you see issues @iphutch

**Comment by iphutch** [comment]
It builds without error, but the indentation on line 26 could be problematic down the road. Please fix with two spaces here and I'll approve right away.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (egernst) [reply CODE_OF_CONDUCT.md]: Ah, I see what you mean now (it should follow the spacing from above).  Sure - will edit them now.

**Comment by grahamwhaley** [comment]
@egernst I think you are getting CI failures as you need to rebase to pick up #116 so we can build with the tip of golang - can you give that a short pls?

**Pull request merged**